source/ranges.tex

```diff
@@ -4867,6 +4867,11 @@
 
 \rSec2[range.counted]{Counted view}
 
+\pnum
+A counted view presents a \libconcept{View} of the elements
+of the counted range\iref{iterator.requirements.general} \range{i}{n}
+for some iterator \tcode{i} and non-negative integer \tcode{n}.
```

Comment:
If you're going to require `n` to be non-negative, should this be `\range{i}{i+n}`?

Reply:
No, it should not. Outside of [algorithm], `i+n` only has well-defined meaning when `i` models `RandomAccessIterator` - which was much of the motivation for defining "counted range" in the first place.

Reply:
Ah, right, this is that newfangled "counted range" thingy.

I don't like this abuse of notation. #2932 filed for that. But that shouldn't block this change.